src/layout.sh

```diff
@@ -14,19 +14,22 @@ BSP_DEFAULT_LAYOUTS="tiled\nmonocle";
 
 # Kill old layout process
 kill_layout() {
```

Comment:
`kill_layout`'s purpose is to kill the layout of a desktop. It doesn't make sense for it to accept a pid. Maybe we can have 2 functions `kill_layout` and `kill_process`